semantic-python/src/Language/Python/Core.hs

```diff
@@ -39,10 +41,10 @@ def n = coerce (Stack.:> n)
 -- a Python ExpressionList. Easier than pattern-matching every time.
 -- TODO: when this is finished, we won't need this pattern, as we'll
 -- handle ExpressionLists the smart way every time.
-pattern SingleIdentifier :: Name -> Py.ExpressionList a
+pattern SingleIdentifier :: Coercible t Text => t -> Py.ExpressionList a
 pattern SingleIdentifier name <- Py.ExpressionList
   { Py.extraChildren =
-    [ Py.Expression (Prj (Py.PrimaryExpression (Prj Py.Identifier { text = Name -> name })))
+    [ Py.Expression (Prj (Py.PrimaryExpression (Prj Py.Identifier { text = coerce -> name })))
```

Comment:
If we’re using this in other parts of the system, then let’s move it to somewhere reasonable for such things—touching the Python-to-Core compiler should never require rebuilding the scope graphing support for Python.

Reply:
I’ll move it to `Language/Python/Patterns.hs` or something

Reply:
Perfect 👍